Makefile

```diff
@@ -32,3 +32,18 @@ testacc: build generate-plugin-cache
 
 .PHONY: check
 check: lint test
+
+.PHONY: legacy-tfstate
+legacy-tfstate:
+	# Generate a 0.12.31 tfstate file for use in replace-provider tests.
+	docker run \
+		--interactive \
+		--rm \
+		--tty \
+		--volume $(shell pwd):/src \
+		--workdir /src/test-fixtures/legacy-tfstate \
+		--entrypoint /bin/sh \
+		hashicorp/terraform:0.12.31 \
+			-c \
+				"terraform init && \
+				terraform apply -auto-approve"
```

Comment:
☝️ This is a convenience utility for generating the 0.12.31 TF state test fixture. Perhaps this is useful in the future? If anything, I hope it offers some transparent visibility on my technique for generating the legacy TF state file.

I'm not offended if you feel this is unwarranted, though, @minamijoyo.

Reply:
I don't think it is required, but you may leave it as is.